### PR TITLE
chore: update `LICENSE`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Bring in line with what we have in the RR repo
https://github.com/remix-run/react-router/blob/main/LICENSE.md

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
-Copyright 2021 Remix Software Inc.
+MIT License
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) Remix Software 2021
```
Comment (brophdawg11) [inline LICENSE.md:3]: I'm no lawyer but the [RR version](https://raw.githubusercontent.com/remix-run/react-router/main/LICENSE.md) of this has a year range.  Do we need that for Remix or does the year indicate the start of the copyright period?  Seems tedious to have to update the range every year.  If it carries forward automatically we may want to remove the end year in RR?  Probably a @mjackson question though...

Comment (MichaelDeBoey) [reply LICENSE.md:3]: > or does the year indicate the start of the copyright period

I thought it was like "we copyright this starting the mentioned year"

Comment (mcansh) [inline LICENSE.md:3]:
```suggestion
Copyright (c) Remix Software 2021
Copyright (c) Shopify Inc. 2022
```

Comment (mcansh) [reply LICENSE.md:3]: @mjackson says starting year is fine

File: LICENSE.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+Copyright (c) Shopify Inc. 2022
```
Comment (mcansh) [inline LICENSE.md:4]:
```suggestion
Copyright (c) Remix Software 2021

Copyright (c) Shopify Inc. 2022
```

that way it appears on 2 lines

Comment (MichaelDeBoey) [reply LICENSE.md:4]:
```suggestion
Copyright (c) Remix Software 2021  
Copyright (c) Shopify Inc. 2022
```

File: LICENSE.md [hunk 3]
```diff
@@ -1,7 +1,22 @@
-Copyright 2021 Remix Software Inc.
+MIT License
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) Remix Software 2021  
+Copyright (c) Shopify Inc. 2022
```
Comment (mjackson) [inline LICENSE.md:4]:
```suggestion
Copyright (c) Remix Software Inc. 2020-2021
Copyright (c) Shopify Inc. 2022-2023
```

**Pull request merged**